HACKTOBERFEST.md

```diff
@@ -0,0 +1,27 @@
+# :pizza: Hacktoberfest 2019 :beer:
```

Comment:
Big link from the top of readme (image maybe?) to this document will be useful for new contributors

Reply:
Good idea, done